app/views/users/show.html.erb

```diff
@@ -1 +1,31 @@
-<h2> here we will more details about user </h2>
+<main>
+    <div class="user-container">
+        <section class="user">
+            <img src=<%= @user.photo %> alt="User Photo" class="user-photo">
+            <div class="user-details">
+                <h2 class="user-name"><%= @user.name%></h2>
+                <h5 class="user-post-count">Number of posts: <%= @user.post_counter %> </h5>
+            </div>
+        </section>
+        <section class="user-bio">
+            <h3>Bio</h3>
+            <p><%= @user.bio %></p>
+        </section>
+        <section class="user-posts">
+            <h3>Recent Posts</h3>
+            <% @user.three_latest_posts.each do |post| %>
+            <a href="<%=@user.id%>/posts/<%=post.id%>" class="user-post-link">
+                <div class="post">
+                    <h4><%= post.title %></h4>
+                    <p><%= truncate(post.text, length: 180) %></p>
+                    <p class="comment-like-count">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+                </div>
+            </a>
+            <% end %>
+            <button class="button">
```

Comment:
- [ ] Kindly inspect your post table while creating your views. This page gives errors due to disparity between `Title`, `Text` and so on that you have. You are using the incorrect attributes syntax. Kindly go through your code, and ensure everything works.

  ![image](https://github.com/AlaaAlsalem/blogaapp/assets/81468675/6dbed826-9298-4dfd-87e2-02472b68cca0)

  This is what your post table looks like.

  ![image](https://github.com/AlaaAlsalem/blogaapp/assets/81468675/a2b03943-1172-42c3-87ba-736aac9ff224)